pkg/cmd/prebuild/add.go

```diff
@@ -36,57 +40,99 @@ var prebuildAddCmd = &cobra.Command{
 			return err
 		}
 
-		gitProviders, res, err := apiClient.GitProviderAPI.ListGitProviders(ctx).Execute()
-		if err != nil {
-			return apiclient_util.HandleErrorResponse(res, err)
-		}
+		// If no arguments and no flags are provided, run the interactive CLI
+		if len(args) == 0 && branchFlag == "" && retentionFlag == 0 &&
```

Comment:
![Screenshot 2024-10-14 at 10 35 07](https://github.com/user-attachments/assets/30424ab1-4b27-4308-b873-b1d44d5c7ce6)

Interactive mode is broken now.

You need to test **all** changes before requesting a review. Please provide a video of how this PR works.

Reply:
- it is fixed now